Tests/interop/net/field/test_field_misc.py

```diff
@@ -16,28 +16,31 @@
 Operations on enum type and its' members
 '''
 #------------------------------------------------------------------------------
-from iptest import *    
-from iptest.assert_util import *
 
+import unittest
 
-add_clr_assemblies("fieldtests", "typesamples")
+from iptest import IronPythonTestCase
 
-if options.RUN_TESTS: #TODO - bug when generating Pydoc
-    from Merlin.Testing.FieldTest import *
-    from Merlin.Testing.TypeSample import *
+class FieldMiscTest(IronPythonTestCase):
```

Comment:
I'm actually thinking about running the tests under CPython. Maybe not under CI, but it would be useful to highlight compatibility issues.